mssql-plugin/src/e2e-test/features/mssql/mssql source/RunTime.feature

```diff
@@ -0,0 +1,197 @@
+  Scenario: Verify User is able to preview and deploy the pipeline when connection arguments are set
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "SQL Server" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Connect plugins: "SQL Server" and "BigQuery" to establish connection
+    Then Navigate to the properties page of plugin: "SQL Server"
+    Then Select dropdown plugin property: "select-jdbcPluginName" with option value: "driverName"
+    Then Replace input plugin property: "host" with value: "host" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "port" for Credentials and Authorization related fields
+    Then Replace input plugin property: "user" with value: "username" for Credentials and Authorization related fields
+    Then Replace input plugin property: "password" with value: "password" for Credentials and Authorization related fields
+    Then Enter key value pairs for plugin property: "connectionArguments" with values from json: "connectionArgumentsList"
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    Then Replace input plugin property: "database" with value: "databaseName"
+    Then Enter textarea plugin property: "importQuery" with value: "selectQuery"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "outputSchema"
+    Then Validate "SQL Server" plugin properties
+    Then Close the Plugin Properties page
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    And Replace input plugin property: "project" with value: "project.id"
+    And Enter input plugin property: "datasetProject" with value: "datasetprojectId"
+    And Enter input plugin property: "dataset" with value: "dataset"
+    And Enter input plugin property: "table" with value: "bqtarget.table"
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    Then Save the pipeline
+    And Preview and run the pipeline
+    And Wait till pipeline preview is in running state
+    And Open and capture pipeline preview logs
+    And Verify the preview run status of pipeline in the logs is "succeeded"
+    And Close the pipeline logs
+    And Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+
+  @MSSQL_SOURCE_TEST @BQ_SINK @Mssql_Required
+  Scenario: To verify the pipeline fails while preview with invalid bounding query setting the required split-By field
```

Comment:
Instead of checking for a failure on Preview, directly Deploy this pipeline and check the error message provided in logs.

Reply:
resolved

Reply:
We are still checking if the pipeline is failed not checking the error message.